### ISSUE TITLE
cifar, the acc of training set is lower than that of test set?

### ISSUE THREAD

**Issue by chuchuhe** [kai-wen-yang/CD-VAE#3]
I ran the original code 'distentangle_cifar.py' and got the below result. 
Is it the same result as yours? Is it correct?
If so, I am confused that why the acc of training set is lower than that of test set.
![微信图片_20220310194956](https://user-images.githubusercontent.com/77535209/157656116-a6eaed19-e3fc-4a5b-83aa-1d971db6a0fe.png)

**Comment by kai-wen-yang** [comment]
> I ran the original code 'distentangle_cifar.py' and got the below result.
> Is it the same result as yours? Is it correct?
> If so, I am confused that why the acc of training set is lower than that of test set.
> ![微信图片_20220310194956](https://user-images.githubusercontent.com/77535209/157656116-a6eaed19-e3fc-4a5b-83aa-1d971db6a0fe.png)

Training uses random augmentation which is difficult to classify, testing does not use data augmentation.
